pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
I don't see any other types that have their own `storage.go`, but I see many that define a custom Strategy.  I think you should use that approach if possible.  See `pkg/registry/secret/strategy.go` for example.  Unless there is some reason it needs to be at the Storage layer.